crates/loans/src/lib.rs

```diff
@@ -343,6 +346,40 @@ pub mod pallet {
         IncentiveReservesReduced(T::AccountId, AssetIdOf<T>, BalanceOf<T>),
     }
 
+    #[pallet::hooks]
+    impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
+        fn on_runtime_upgrade() -> frame_support::weights::Weight {
+            let max_exchange_rate = crate::MaxExchangeRate::<T>::get();
+            if max_exchange_rate.is_zero() {
+                crate::MaxExchangeRate::<T>::put(Rate::from_inner(DEFAULT_MAX_EXCHANGE_RATE));
+            }
+            let min_exchange_rate = crate::MinExchangeRate::<T>::get();
+            if min_exchange_rate.is_zero() {
+                crate::MinExchangeRate::<T>::put(Rate::from_inner(DEFAULT_MIN_EXCHANGE_RATE));
+            }
+            T::DbWeight::get().reads_writes(0, 2)
```

Comment:
Nit: should be this, but it doesn't matter a ton
```suggestion
            T::DbWeight::get().reads_writes(2, 2)
```